pulsar-metadata/src/test/java/org/apache/pulsar/metadata/CounterTest.java

```diff
@@ -95,6 +95,8 @@ public void testCounterDoesNotAutoReset(String provider, Supplier<String> urlSup
         assertNotEquals(l1, l4);
         assertNotEquals(l2, l4);
         assertNotEquals(l3, l4);
+
+        store2.close();
```

Comment:
please use try/finally block to ensure the store is closed even if the test won't pass

Reply:
already fixed